app/src/main/java/com/kickstarter/libs/utils/AnimationUtils.kt

```diff
@@ -20,4 +26,52 @@ object AnimationUtils {
     animation.fillAfter = true
     return animation
   }
+
+  @JvmOverloads
+  fun fadeInAndScale(view: View, duration: Long = 500L, startDelay: Long = 500L,
+                     interpolator: Interpolator = AccelerateDecelerateInterpolator()): ObjectAnimator {
+    val scaleX = PropertyValuesHolder.ofFloat("scaleX", 0f, 1f)
```

Comment:
Should I make these strings constants?

Reply:
Yes seeing that we're using them in many places. Good call 😉